### PR TITLE
[zh-cn]: update the translation of `<big>` element

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/html/element/big/index.md [hunk 1]
```diff
@@ -1,55 +1,82 @@
 ---
-title: <big>
+title: <big>：大文本元素
```
Comment (T34-active) [inline files/zh-cn/web/html/element/big/index.md:2]:
```suggestion
title: <big>：较大的文本元素
```

File: files/zh-cn/web/html/element/big/index.md [hunk 2]
```diff
@@ -1,55 +1,82 @@
...
 slug: Web/HTML/Element/big
+l10n:
+  sourceCommit: 942a529383ee7ee3996fb234187641c08935f3ff
 ---
 
-{{deprecated_header}}
+{{HTMLSidebar}}{{deprecated_header}}
 
-## 概要
+**`<big>`** [HTML](/zh-CN/docs/Web/HTML) 已弃用的元素会使其中包裹的文字渲染为比周围文本大一级的字体大小（例如将 `medium` 变为 `large`）。字体大小的最大值受限于浏览器的允许范围。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/big/index.md:10]:
```suggestion
已弃用的 **`<big>`** [HTML](/zh-CN/docs/Web/HTML) 元素使其中包裹的文字会以比周围文本大一级的字体大小（例如将 `medium` 变为 `large`）渲染。字体大小的最大值受限于浏览器的允许范围。
```

File: files/zh-cn/web/html/element/big/index.md [hunk 3]
```diff
@@ -1,55 +1,82 @@
...
 
-The HTML Big Element (`<big>`) 会使字体加大一号（例如从小号 (small) 到中号 (medium)，从大号 (large) 到加大 (x-large)），最大不超过浏览器的最大字体。
-
-> **备注：** 由于它是纯显示性的，该元素在[HTML5](/zh-CN/docs/Web/Guide/HTML/HTML5)中已经被移除，不应当再使用。取而代之，网页开发者应当使用 CSS 属性。
+> **警告：** 该元素已被从规范中移除，不应再继续使用。请改用 CSS {{cssxref("font-size")}} 属性来调整字体大小。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/big/index.md:12]:
```suggestion
> **警告：** 该元素已从规范中移除，不应继续使用。请改用 CSS {{cssxref("font-size")}} 属性来调整字体大小。
```

File: files/zh-cn/web/html/element/big/index.md [hunk 4]
```diff
@@ -1,55 +1,82 @@
...
 
 ## 属性
 
-只包含 [全局属性](/zh-CN/docs/HTML/global_attributes)
+这个元素除了[全局属性](/zh-CN/docs/Web/HTML/Global_attributes)之外，没有其他的属性。
+
+## 示例
+
+在此处，我们展示了 `<big>` 标签的示例，然后紧随其后展示一个现代 CSS 语法实现相同效果的例子。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/big/index.md:20]:
```suggestion
在此处，我们展示了 `<big>` 的用法，然后紧随其后展示一个现代 CSS 语法实现相同效果的例子。
```

File: files/zh-cn/web/html/element/big/index.md [hunk 5]
```diff
@@ -1,55 +1,82 @@
...
+{{EmbedLiveSample("使用_CSS_font-size", 640, 60)}}
 
 ## DOM 接口
 
-该元素实现了 {{domxref('HTMLElement')}}接口。
+该元素实现了{{domxref('HTMLElement')}} 接口。
 
-> **备注：** 直到 Gecko 1.9.2, Firefox 为该元素实现了{{domxref('HTMLSpanElement')}}接口。
+<!-- ## Technical summary -->
+
```
Comment (yin1999) [inline files/zh-cn/web/html/element/big/index.md:69]: 无需保留注释

```suggestion
```

File: files/zh-cn/web/html/element/big/index.md [hunk 6]
```diff
@@ -1,55 +1,82 @@
...
-### 结果
+#### 结果
 
-This is the first sentence. This whole sentence is in bigger letters.
+{{EmbedLiveSample("使用_CSS_font-size", 640, 60)}}
 
 ## DOM 接口
 
-该元素实现了 {{domxref('HTMLElement')}}接口。
+该元素实现了{{domxref('HTMLElement')}} 接口。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/big/index.md:66]:
```suggestion
该元素实现了 {{domxref('HTMLElement')}} 接口。
```

**Comment by T34-active** [comment]
@yin1999 Thank you!

**Review by yin1999 — APPROVED** [review]
Thanks @T34-active

**Pull request merged**